diff --git a/src/common/maptypes.h b/src/common/maptypes.h
index eb375f3..6a4ac1e 100644
--- a/src/common/maptypes.h
+++ b/src/common/maptypes.h
@@ -2044,7 +2044,7 @@ void registerMapMetaTypes();
  * QTypeInfo<TYPE>::isRelocatable * std::is_copy_constructible_v<TYPE > */
 // static_assert(std::is_copy_constructible<map::MapBase>);
 
-Q_DECLARE_TYPEINFO(map::MapBase, Q_RELOCATABLE_TYPE);
+//Q_DECLARE_TYPEINFO(map::MapBase, Q_RELOCATABLE_TYPE);
 Q_DECLARE_TYPEINFO(map::MapAirport, Q_RELOCATABLE_TYPE);
 Q_DECLARE_TYPEINFO(map::MapAirportMsa, Q_RELOCATABLE_TYPE);
 Q_DECLARE_TYPEINFO(map::MapAirspace, Q_RELOCATABLE_TYPE);
