diff --git a/src/search/searchbasetable.cpp b/src/search/searchbasetable.cpp
index e8fd59b..e4408e8 100644
--- a/src/search/searchbasetable.cpp
+++ b/src/search/searchbasetable.cpp
@@ -568,7 +568,7 @@ void SearchBaseTable::connectSearchWidgets()
     }
     else if(col->getCheckBoxWidget() != nullptr)
     {
-      connect(col->getCheckBoxWidget(), &QCheckBox::stateChanged, this, [col, this](int state) {
+      connect(col->getCheckBoxWidget(), &QCheckBox::checkStateChanged, this, [col, this](int state) {
         controller->filterByCheckbox(col, state, col->getCheckBoxWidget()->isTristate());
         updateButtonMenu();
         editStartTimer();
@@ -606,7 +606,7 @@ void SearchBaseTable::connectSearchWidgets()
   if(minDistanceWidget != nullptr && maxDistanceWidget != nullptr && distanceDirWidget != nullptr && distanceCheckBox != nullptr)
   {
     // If all distance widgets are present connect them
-    connect(distanceCheckBox, &QCheckBox::stateChanged, this, &SearchBaseTable::distanceSearchStateChanged);
+    connect(distanceCheckBox, &QCheckBox::checkStateChanged, this, &SearchBaseTable::distanceSearchStateChanged);
 
     connect(minDistanceWidget, QOverload<int>::of(&QSpinBox::valueChanged), this, [this, distanceDirWidget, maxDistanceWidget](int value) {
       controller->filterByDistanceUpdate(
