diff --git a/src/logbook/logdatadialog.ui b/src/logbook/logdatadialog.ui
index 4f8e2a6..a46f0ec 100644
--- a/src/logbook/logdatadialog.ui
+++ b/src/logbook/logdatadialog.ui
@@ -249,9 +249,6 @@
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
-                <property name="timeSpec">
-                 <enum>Qt::UTC</enum>
-                </property>
                </widget>
               </item>
               <item row="2" column="0">
@@ -317,9 +314,6 @@
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
-                <property name="timeSpec">
-                 <enum>Qt::LocalTime</enum>
-                </property>
                </widget>
               </item>
               <item row="0" column="1">
@@ -432,9 +426,6 @@
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
-                <property name="timeSpec">
-                 <enum>Qt::LocalTime</enum>
-                </property>
                </widget>
               </item>
               <item row="3" column="1">
@@ -445,9 +436,6 @@
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
-                <property name="timeSpec">
-                 <enum>Qt::UTC</enum>
-                </property>
                </widget>
               </item>
               <item row="0" column="1">
