diff --git a/src/routeexport/routeexportdialog.ui b/src/routeexport/routeexportdialog.ui
index 802563c..2baec90 100644
--- a/src/routeexport/routeexportdialog.ui
+++ b/src/routeexport/routeexportdialog.ui
@@ -54,9 +54,6 @@
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
-       <property name="timeSpec">
-        <enum>Qt::UTC</enum>
-       </property>
       </widget>
      </item>
      <item row="20" column="3">
@@ -201,9 +198,6 @@
        <property name="currentSectionIndex">
         <number>0</number>
        </property>
-       <property name="timeSpec">
-        <enum>Qt::UTC</enum>
-       </property>
       </widget>
      </item>
      <item row="12" column="0">
