diff --git a/src/webapi/abstractlnmactionscontroller.h b/src/webapi/abstractlnmactionscontroller.h
index a2a9db2..e129a92 100644
--- a/src/webapi/abstractlnmactionscontroller.h
+++ b/src/webapi/abstractlnmactionscontroller.h
@@ -62,12 +62,12 @@ class AbstractLnmActionsController :
   public AbstractActionsController
 {
   Q_OBJECT
-  Q_ENUMS(AirportQueryType)
 
 public:
   Q_INVOKABLE AbstractLnmActionsController(QObject *parent, bool verboseParam, AbstractInfoBuilder *infoBuilder);
   virtual ~AbstractLnmActionsController() override;
   enum AirportQueryType {SIM, NAV};
+  Q_ENUM(AirportQueryType)
 
 protected:
   // Main LNM objects
