diff --git a/src/common/formatter.cpp b/src/common/formatter.cpp
index 4e562b1..e40bffe 100644
--- a/src/common/formatter.cpp
+++ b/src/common/formatter.cpp
@@ -60,7 +60,7 @@ QString formatTimeZoneTextShort(const QTimeZone& timezone, const QDateTime& offs
     const QString offset = formatter::formatTimeZoneOffset(timezone.standardTimeOffset(offsetRefTime));
     QLocale::Territory territory = timezone.territory();
     QString territoryStr = QLocale::territoryToString(territory);
-    if(territory != QLocale::AnyTerritory && !territoryStr.isEmpty() && territory != QStringLiteral("Default"))
+    if(territory != QLocale::AnyTerritory && !territoryStr.isEmpty() && territoryStr != QStringLiteral("Default"))
       timeZoneText = atools::strJoin({territoryStr, offset}, separator);
     else
       timeZoneText = offset;
